uk.co.compucorp.civicrm.hrcontactactionsmenu/js/contactactions.js

```diff
@@ -14,8 +14,8 @@
 
       CRM
         .confirm({
-          'title': 'Confirmation dialog',
-          'message': 'Are you sure you want to continue?'
+          'title': 'Confirm',
+          'message': 'Are you sure you want to delete the user account?'
```

Comment:
Was this text confirmed with PM? I think it would be better to have "Are you sure you want to delete *this* user account?"

Reply:
Yes it comes from [here](https://compucorp.atlassian.net/wiki/spaces/PCHR/pages/121277611/Contact+page+actions+-+UX+improvements)